docs/topics/debug-coroutines-with-idea.md

```diff
@@ -100,3 +100,17 @@ The tutorial assumes you have prior knowledge of the [coroutines](coroutines-gui
     * The third coroutine is calculating the value of `b` – it has the **RUNNING** status.
 
 Using IntelliJ IDEA debugger, you can dig deeper into each coroutine to debug your code.
+
+### Debug suspend functions
```

Comment:
```suggestion
### Optimised-out variables
```
I think this reflects the idea of the subsection better.

Reply:
True, thanks!